src/errors/BaseError.ts

```diff
@@ -0,0 +1,11 @@
+class BaseError extends Error {
+  status: number
+  message: string
+  constructor (status: number, message: string) {
+    super(message)
+    this.status = status
+    this.message = message
```

Comment:
Also, instead of doing `this.status = status` for each property, you could use public keyword, and it will be he same -> declaring + assigning. 

```
  constructor (message: string, public status: number) {
    super(message)
  }
```

Reply:
61484fc9f1f6b2093aedd924